bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh

```diff
@@ -166,7 +166,7 @@ export PGBOUNCER_CLIENT_IDLE_TIMEOUT="${PGBOUNCER_CLIENT_IDLE_TIMEOUT:-}"
 # PostgreSQL backend settings
 export POSTGRESQL_USERNAME="${POSTGRESQL_USERNAME:-postgres}"
 export POSTGRESQL_PASSWORD="${POSTGRESQL_PASSWORD:-}"
-export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-postgres}"
+export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-${PGBOUNCER_DATABASE:-postgres}}"
```

Comment:
As PGBOUNCER_DATABASE is already set, you can use this

```suggestion
export POSTGRESQL_DATABASE="${POSTGRESQL_DATABASE:-${PGBOUNCER_DATABASE}}"
```

What do you think?

Reply:
yes, I think it's better